lib/flipper/adapter_builder.rb

```diff
@@ -3,7 +3,7 @@ module Flipper
   #
   #   adapter = Flipper::AdapterBuilder.new do
   #     use Flipper::Adapters::Strict
-  #     use Flipper::Adapters::Memoizer
+  #     use Flipper::Adapters::Memoizable
```

Comment:
This is unrelated to the PR but I thought I added this small typo fix. I'm happy to create another PR for this if that's more appropriate.